marlowe-runtime/doc/deployment.md

```diff
@@ -34,7 +34,6 @@ $ nix run .#marlowe-chain-sync -- --help
 $ nix run .#marlowe-indexer -- --help
 $ nix run .#marlowe-sync -- --help
 $ nix run .#marlowe-tx -- --help
-$ nix run .#marlowe -- --help
```

Comment:
We don't have a `marlowe` executable. Maybe this has been renamed?

Reply:
Actually, it was renamed to `marlowe-runtime-cli`, but we're retiring it.